content/components/overlay.mdx

```diff
@@ -1,15 +1,23 @@
 ---
 title: Overlay
 description: Overlay components codify design patterns related to floating surfaces such as dialogs and menus.
+reactId: overlay
 railsIds:
-- Primer::Alpha::Overlay
+Primer::Alpha::Overlay
 ---
 
 import ComponentLayout from '~/src/layouts/component-layout'
-import {Text} from '@primer/react'
+import {Link, Text} from '@primer/react'
+import {Link as GatsbyLink} from 'gatsby'
 export default ComponentLayout
 
+### Usage
+
+Overlay is an internal component and is intended to be used as a private API, composing other components. It is not intended to be used on its own. 
+
+The overlay component handles all behaviors needed by overlay UIs as well as the common styles that all overlays should have. Overlay is the base component for many of our overlay-type components. Overlays use shadows to express elevation.
+
 <Note variant="warning">
-  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Work in progress</Text>
-  We are currently working on general guidance for this component.
+  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Guidelines for this component are in progress</Text>
```

Comment:
It looks like sometimes we put this `Note` under a "Usage" heading and sometimes we don't. Let's keep it consistent.

Personally, I prefer having it outside of the "Usage" heading, but I'm fine with whatever as long as it's consistent.